<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mytaskswidgetitem</class>
 <widget class="QWidget" name="mytaskswidgetitem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>130</width>
    <height>95</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>130</width>
    <height>95</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>30000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mytaskswidgetitem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 10px</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="bgFrame">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(186, 186, 186)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QFrame" name="header">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0,0,0)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="projectName">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white</string>
           </property>
           <property name="text">
            <string>Project Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="phaseName">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Phase Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Hours:</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="hours">
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0,0,0,0)</string>
                </property>
                <property name="text">
                 <string>20</string>
                </property>
               </widget>
              </item>
             </layout>
             <zorder>due</zorder>
             <zorder>groupBox</zorder>
             <zorder>groupBox</zorder>
             <zorder>hours</zorder>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Due:</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="due">
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 0, 4,); background-color: rgba(0,0,0,0)</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>10/20/20</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
